File: map2d/src/main/java/net/ponderingprogrammer/map2d/CellGraph.java
package net.ponderingprogrammer.map2d;

import java.util.HashSet;

public class CellGraph<T> {
    private HashSet<Cell<T>> nodes = new HashSet<>();
    private HashSet<CellGraphEdge<T>> edges = new HashSet<>();

    public HashSet<Cell<T>> getNodes() {
        return nodes;
    }

    public HashSet<CellGraphEdge<T>> getEdges() {
        return edges;
    }
}

File: projects/wpf/NProcGen/src/PonderingProgrammer.Map2d/CellGraph.cs
using System.Collections.Generic;

namespace PonderingProgrammer.Map2d
{
    public class CellGraph<T>
    {
        public HashSet<Cell<T>> Nodes { get; } = new HashSet<Cell<T>>();
        public HashSet<CellGraphEdge<T>> Edges { get; } = new HashSet<CellGraphEdge<T>>();
    }
}
